### DISCUSSION TITLE
小程序lite版本偶现播放两次，并且画质差

### DISCUSSION THREAD

**Discussion by liuzixinkim** [Tencent/libpag#1610] [Q&A]
【版本信息】
libpag-lite-miniprogram的版本： 0.0.6

【平台信息】
微信小程序

【预期的表现】
希望点击不同的按钮的时候，序列播放动效，播放完一个再继续播放下一个

【实际的情况】
但是在实际使用中发现，有两个问题：

第一个偶现的问题是：在连续播放过程中，偶现某个动效会播放两次，且并不会触发onAnimationRepeat事件监听函数，在onAnimationEnd监听函数里也没有监听到第一次播放结束，并且给setRepeatCount设置为1也没办法解决。

第二个问题是：动效的画质低，同样的文件资源，用libpag-miniprogram版本播放画质就很好，但是用lite版本，画质就差，这个可以怎么解决一下呢。

【Demo及附件】
[pag-test.zip](https://github.com/Tencent/libpag/files/12143105/pag-test.zip)

**Comment by zhalice2011** [comment]
你好, 收到反馈, 我先本地复现一下

另外 https://rt-festatic.oss-cn-beijing.aliyuncs.com/live/img/fans/1_bmp.pag 这个文件好像被删除了

**Reply by liuzixinkim** [reply]
不好意思，之前那个文件删除了。可以用下面这些文件播放一下
https://rt-festatic.oss-cn-beijing.aliyuncs.com/live/img/pag-file/1_bmp.pag
https://rt-festatic.oss-cn-beijing.aliyuncs.com/live/img/pag-file/data_video.pag
https://rt-festatic.oss-cn-beijing.aliyuncs.com/live/img/pag-file/jisha.pag

**Comment by Alex-Archer** [comment]
请问你处理了吗？我现在也遇到这样的情况，能显示就是动图很糊
